triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/LoadTesting.scala

```diff
@@ -135,7 +163,7 @@ final class InFlightLoadTesting extends LoadTesting {
 
   override protected def triggerRunnerConfiguration: TriggerRunnerConfig =
     super.triggerRunnerConfiguration
-      .copy(inFlightCommandBackPressureCount = 20, inFlightCommandOverflowCount = 200)
+      .copy(inFlightCommandBackPressureCount = 20, inFlightCommandOverflowCount = 40)
```

Comment:
naive question, why are we reducing this parameter ?

Reply:
More to do with testing speed here.

Goal of the test is to ensure that too many in-flight commands can cause an overflow. For this reason, having a lower number allows the test to overflow sooner than with a number of 200. With throttling set by default at 100 messages per 5 seconds, it takes around 10 seconds to emit enough command submissions to overflow the in-flight counts. Setting at 40, we can achieve the same goal in 2 seconds.